src/libssh.win-x64/context-dir/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM fedora
+
+# Install dependencies
+RUN dnf install -y mingw64-openssl 'dnf-command(download)' cpio xz cmake \
+                   mingw64-gcc-c++ binutils
+
+WORKDIR /root
+
+# Download libssh rpm sources.
+RUN dnf download libssh --source && \
+    rpm2cpio libssh-*.src.rpm  | cpio -idmv && \
+    tar -xf libssh-*.tar.xz && \
+    ls && \
+    mv $(find . -maxdepth 1 -name 'libssh-*' -type d) libssh
+
+# Build and install libssh.
+RUN mkdir libssh/build && \
+    cd libssh/build && \
+    mingw64-cmake .. && \
+    make -j$(nproc) && \
+    make install
```

Comment:
@omajid I'm getting the Fedora libssh sources here and then use `mingw64-cmake` to build them for `win-x64`. Do you have suggestions on how I can do this better?

I'm just building stuff at this point, I haven't verified these dlls actually work on Windows.

You can also look at this later and send a PR to improve how things get built.